app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -0,0 +1,31 @@
+<%#
+kind: iPXE
+model: ProvisioningTemplate
+name: iPXE intermediate script
+snippet: false
+-%>
+#!ipxe
+
+# Skips booting from network and continues booting from next device
+exit
+
+#!ipxe
+# Intermediate iPXE script to report MAC address to Foreman
+isset ${net0/mac} || goto no_nic
+dhcp net0 || goto net1
+chain <%= foreman_url('iPXE') %>?mac=${net0/mac} || goto net1
+exit 0
+
+:net1
+isset ${net1/mac} || goto no_nic
+dhcp net1 || goto net2
+chain <%= foreman_url('iPXE') %>?mac=${net1/mac} || goto net2
+exit 0
+
+:net1
+# Create as many copies as necessary (the example will work up to 2 NICs)
+
+:no_nic
+echo Failed to chainload from any network interface
+sleep 30
+exit 1
```

Comment:
Since this is ERB now, can you actually use this:

https://github.com/theforeman/foreman_bootdisk/blob/master/app/views/foreman_bootdisk/host.erb#L17

Reply:
What I mean, this was just for two NICs, server have often 4-8 or more, so just generate this with ERB up to 32 NICs.